packages/subgraph/CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
+- Changed attribute of `proposalId` in all proposal entities to `pluginProposalId`.
```

Comment:
If it's a breaking change to external users, I would add some `BREAKING` mention

Reply:
Shouldn't this be reflected through the increment of the `MAJOR` version number according to the [semantic versioning conventions](https://semver.org/spec/v2.0.0.html)?

> Given a version number MAJOR.MINOR.PATCH, increment the:
> 1. MAJOR version when you make incompatible API changes
> 2. MINOR version when you add functionality in a backwards compatible manner
> 3. PATCH version when you make backwards compatible bug fixes
>
> Additional labels for pre-release and build metadata are available as extensions to the MAJOR.MINOR.PATCH format.